ehitchhiking-ui/src/app/shared/modals/profile-modal/cars-info.service.ts

```diff
@@ -0,0 +1,34 @@
+import {Injectable} from '@angular/core';
+import {FormGroup, Validators, FormBuilder} from '@angular/forms';
+import {Car} from '../../models/car';
+
+@Injectable()
+export class CarsInfoService {
+	constructor() {}
+
+	public toFormGroup(cars: Car[], formBuilder: FormBuilder): FormGroup {
+		const group: any = {};
```

Comment:
any?

Reply:
which type should it be? FormGroup[], FormArray, AbstractConrol[] don't work, because like this i can't make a FormGroup from variable group event with the converting

Reply:
Proper type can be found in FormGroup constructor signature

Reply:
You can find proper type in formBuilder and FormGroup constructor signature.

Reply:
can not resolve this issue, added comment